src/core/connection.c

```diff
@@ -3256,7 +3256,11 @@ QuicConnQueueRecvPackets(
         QUIC_OPERATION* ConnOper =
             QuicConnAllocOperation(Connection, QUIC_OPER_TYPE_FLUSH_RECV);
         if (ConnOper != NULL) {
-            QuicConnQueueOper(Connection, ConnOper);
+            if (!Connection->State.HandshakeConfirmed) {
+                QuicConnQueuePriorityOper(Connection, ConnOper);
```

Comment:
Thanks for the contribution.
Could you please add a comment here summarizing why we need a priority operation before the handshake is confirmed?